docs/mp/messaging/01_reactive-messaging.adoc

```diff
@@ -0,0 +1,159 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2020 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Reactive Messaging
+:description: Helidon MicroProfile Reactive Messaging
+= :keywords: helidon, messaging, reactive, streams
+
+Since Helidon 2.0
+
+== Reactive Messaging
+
+MicroProfile Reactive Messaging provides means for CDI beans to be able produce, consume or process messages over Reactive Streams.
+Such messaging bean is expected to be either in `ApplicationScoped` or `Dependent` scope.
+Messages are managed by methods annotated by `@Incoming` and `@Outgoing`, invocation is always driven by messaging core either at assembly time or for every message coming from the stream.
```

Comment:
Messages are managed by methods annotated by `@Incoming` and `@Outgoing' and the invocation is always driven by message core - either at assembly time, or for every message coming from the stream.